packages/fetchai/connections/p2p_libp2p/libp2p_node/dht/dhtpeer/dhtpeer.go

```diff
@@ -1650,13 +1677,13 @@ func (dhtPeer *DHTPeer) handleAeaAddressStream(stream network.Stream) {
 		}
 	}
 
-	status := &acn.Status{
-		Code: acn.Status_ERROR_GENERIC,
+	status := &acn.StatusBody{
+		Code: acn.ERROR_GENERIC,
```

Comment:
`AGENT_RECORD_NOT_FOUND`?

Reply:
set to `ERROR_UNKNOWN_AGENT_ADDRESS`
not going add new error statuses at the moment